rust/src/proofs/api.rs

```diff
@@ -192,7 +192,7 @@ pub unsafe extern "C" fn fil_fauxrep2(
             }
             Err(err) => {
                 response.status_code = FCPResponseStatus::FCPUnclassifiedError;
-                response.error_msg = rust_str_to_c_str(format!("{:?}", err));
+                response.error_msg = rust_str_to_c_str(format!("{:?}", err)).unwrap();
```

Comment:
Ideally we'd either:

1. Store these as "byte" strings (ptr & length) instead of C strings, given that we don't really care about C.
2. Escape null bytes. I.e., have a `rust_err_to_c` function that converts errors to strings, escaping null bytes, special characters, etc.

Reply:
Or is the goal here to use the `fil_Result` type?

Reply:
this will go away soon